### PR TITLE
Fix annotation for cronjob when is manually triggered

### LINKED ISSUE #4681: Triggering a cronjob from lens does not set annotation `cronjob.kubernetes.io/instantiate manual` on the job
**Describe the bug**
When you run `kubectl create job --from=cronjob/mycronjob myjob`, an annitation is set on the newly created job: `cronjob.kubernetes.io/instantiate: manual`. When you use "trigger" functionality on a CronJob from Lens, the newly created job does not have that annotation.

**To Reproduce**
1. Go to CronJobs section in Lens
2. Click on the three vertical dots and then on Trigger, 
3. Give your job a name and click Trigger.
4. Examine newly created job, observe that the annothation is absent

**Expected behavior**
`cronjob.kubernetes.io/instantiate: manual` annotation is present on the new job.

**Environment (please complete the following information):**
- Lens Version: 5.3.3
- OS: Windows 10

### PR DESCRIPTION
**Pull request by krlosmederos**
Signed-off-by: Carlos René Mederos Arias <krlosmederos@gmail.com>

Fixes #4681

It adds a default annotation for cronjobs that are instantiated manually.

### CODE REVIEW

File: src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ export class CronJobTriggerDialog extends Component<CronJobTriggerDialogProps> {
       }, {
         spec: cronjobDefinition.spec.jobTemplate.spec,
         metadata: {
+          annotations: { "cronjob.kubernetes.io/instantiate": "manual" } as { [annotation: string]: string },
```
Comment (Nokel81) [inline src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx]: This cast should not be necessary. Is it?

Comment (krlosmederos) [reply src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx]: Without the cast, `make build` throws an error (`make lint` and `make test` pass ok). 

CleanShot 2022-03-07 at 13 08 05

Comment (Nokel81) [reply src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx]: I see, the problem is actually on line `92` and the `as KubeObjectMetadata` cast should be removed.

Comment (krlosmederos) [reply src/renderer/components/+workloads-cronjobs/cronjob-trigger-dialog.tsx]: Thanks, It works. I am amending it.

**Review by Nokel81 — APPROVED** [review]

**Comment by Nokel81** [comment]
Thanks for the PR

**Review by jakolehm — APPROVED** [review]

**Pull request merged**